web/scenes/Portal/Teams/TeamId/Apps/AppId/Configuration/AppStoreRefactored/ImageForm/ImageUploadField.tsx

```diff
@@ -139,12 +144,17 @@ export const ImageUploadField = (props: ImageUploadFieldProps) => {
       } catch (error) {
         console.error("error uploading image:", error);
 
+        if (error instanceof Error && error.name === "AbortError") {
+          toast.error("Upload was cancelled", { autoClose: 5000 });
+          return;
+        }
+
         if (error instanceof ImageValidationError) {
```

Comment:
I removing the comment can just make this inverted so it doesn't look weird
```suggestion
        if (!(error instanceof ImageValidationError)) {
```